shell/tests/chain_test.rs

```diff
@@ -693,13 +697,49 @@ fn process_bootstrap_level1324_and_mempool_for_level1325(
         }
     }
 
+    // print dir size
+    print_dir(3, &root_dir_temp_storage_path, true)?;
+    print_dir(3, &root_context_db_path, true)?;
     // stop nodes
     drop(mocked_peer_node);
     drop(node);
 
     Ok(())
 }
 
+fn print_dir<P: AsRef<Path>>(
+    depth: u64,
+    path: P,
+    human_format: bool,
+) -> Result<(), failure::Error> {
+    let mut options = DirOptions::new();
+    options.depth = depth;
+    let dir_content = get_dir_content2(path, &options)?;
+    for directory in dir_content.directories {
+        let dir_size = if human_format {
+            human_readable(get_size(&directory)?)
+        } else {
+            get_size(&directory)?.to_string()
+        };
+        println!("{} {}", dir_size, &directory); // print directory path and size
```

Comment:
@mambisi 
reverse order please: directory -> size,
it will be easier to compare

Reply:
ohk